wagtail/snippets/templates/wagtailsnippets/snippets/edit.html

```diff
@@ -1,9 +1,14 @@
 {% extends "wagtailadmin/base.html" %}
 {% load wagtailadmin_tags i18n %}
-{% block titletag %}{% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Editing {{ snippet_type_name }} - {{ instance }}{% endblocktrans %}{% endblock %}
+{% block titletag %}{% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Editing {{ snippet_type_name }} - {{ object }}{% endblocktrans %}{% endblock %}
+{% block bodyclass %}page-editor {% if object.live %}page-is-live{% endif %} model-{{ model_opts.model_name }} {% endblock %}
```

Comment:
I just checked, turns out it's only used for auto-populating the slug on the page editor (been there since forever). I've removed this instead, it should be safe.